gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/BundleMapperService.java

```diff
@@ -193,4 +193,4 @@ private RCMRMT030101UK04EhrExtract getEhrExtract(RCMRIN030000UK06Message xmlMess
     private <T extends DomainResource> void addEntry(Bundle bundle, T resource) {
         bundle.addEntry(new BundleEntryComponent().setResource(resource));
     }
-}
+}
```

Comment:
You could leave that empty line at the end of the file :D

Reply:
Done :)

Reply:
I mean the last line, not the line before the last `}` :D